ctypesgen/printer_python/preamble/3_2.py

```diff
@@ -389,7 +389,7 @@ def ReturnString(obj, func=None, arguments=None):
 # Non-primitive return values wrapped with UNCHECKED won't be
 # typechecked, and will be converted to c_void_p.
 def UNCHECKED(type):
-    if hasattr(type, "_type_") and isinstance(type._type_, bytes) and type._type_ != "P":
+    if hasattr(type, "_type_") and isinstance(type._type_, str) and type._type_ != "P":
```

Comment:
Are you going to cherry-pick this commit and merge this fix?  Perhaps with a test that demonstrates it working?